blocks/blogsearch/blogsearch.js

```diff
@@ -1,15 +1,69 @@
 import {
-  div, form, input, i,
+  a, div, form, input, i,
```

Comment:
Looking at it now, I realise we don't really need the blog search as a separate block and we could move the JS and CSS in the blognav directly. This would save us 2 round trip requests to retrieve the block JS and CSS.

Reply:
OK, I'll throw these together.

Reply:
resolved in fc63bfe